_data/cloud.yml

```diff
@@ -83,6 +83,13 @@ websites:
       software: Yes
       doc: https://devcenter.heroku.com/articles/two-factor-authentication
 
+    - name: Hostiso
+      url: https://hostiso.com/
+      img: hostiso.png
+      tfa: Yes
+      software: Yes
+      doc: https://hostiso.com/speed-security/
```

Comment:
@trance-with-me since the `doc` link doesn't mention 2fa and isn't required, could you remove it?